packages/ionic/src/commands/deploy/configure.ts

```diff
@@ -22,11 +22,19 @@ For Cordova projects this is not implemented because it is better to reinstall t
       `,
       exampleCommands: [
         '',
-        '--app-id="abcd1234" --channel-name="Master" --update-method="background"',
+        '--app-id=abcd1234 --channel-name=Master --update-method=background',
         '--max-store=2 --min-background-duration=30',
-        '--app-id="abcd1234" --channel-name="Master" --update-method="background" --max-store=2 --min-background-duration=30',
+        '--app-id=abcd1234 --channel-name=Master --update-method=background --max-store=2 --min-background-duration=30',
+        'android',
+        'ios',
       ],
       options: this.commandOptions,
+      inputs: [
+        {
+          name: 'platform',
+          summary: `The native platform (e.g. ${input('ios | android')})`,
```

Comment:
`${['ios', 'android'].map(v => input(v)).join(', ')}` is what we do in other places when listing common values